composer.json

```diff
@@ -31,50 +31,49 @@
         "ext-json": "*",
         "ext-simplexml": "*",
         "api-platform/core": "^3.4.2",
-        "babdev/pagerfanta-bundle": "^3.7 || ^4.0",
-        "behat/transliterator": "^1.3",
-        "doctrine/collections": "^1.6",
+        "babdev/pagerfanta-bundle": "^4.4",
+        "behat/transliterator": "^1.5",
+        "doctrine/collections": "^1.8",
         "doctrine/common": "^3.2",
-        "doctrine/dbal": "^3.0",
-        "doctrine/doctrine-bundle": "^2.3.1",
-        "doctrine/doctrine-migrations-bundle": "^3.0.1",
-        "doctrine/event-manager": "^1.1 || ^2.0",
-        "doctrine/inflector": "^1.4 || ^2.0",
-        "doctrine/migrations": "^3.5.5",
-        "doctrine/orm": "^2.13",
-        "doctrine/persistence": "^2.3 || ^3.0",
-        "egulias/email-validator": "^3.1",
-        "enshrined/svg-sanitize": "^0.16",
-        "fakerphp/faker": "^1.10",
-        "friendsofphp/proxy-manager-lts": "^1.0.7",
+        "doctrine/dbal": "^3.9",
+        "doctrine/doctrine-bundle": "^2.13",
+        "doctrine/doctrine-migrations-bundle": "^3.3",
+        "doctrine/event-manager": "^2.0",
+        "doctrine/inflector": "^2.0",
+        "doctrine/migrations": "^3.8",
+        "doctrine/orm": "^2.18",
+        "doctrine/persistence": "^3.3",
+        "egulias/email-validator": "^4.0",
+        "enshrined/svg-sanitize": "^0.20",
+        "fakerphp/faker": "^1.23",
+        "friendsofphp/proxy-manager-lts": "^1.0",
         "gedmo/doctrine-extensions": "^3.17.1",
-        "guzzlehttp/guzzle": "^6.5 || ^7.0",
+        "guzzlehttp/guzzle": "^7.9",
         "guzzlehttp/psr7": "^2.5",
-        "knplabs/gaufrette": "^0.10 || ^0.11",
-        "knplabs/knp-gaufrette-bundle": "^0.8 || ^0.9",
-        "knplabs/knp-menu": "^3.1",
-        "knplabs/knp-menu-bundle": "^3.0",
-        "laminas/laminas-stdlib": "^3.3.1",
+        "knplabs/gaufrette": "^0.11",
+        "knplabs/knp-gaufrette-bundle": "^0.9",
+        "knplabs/knp-menu": "^3.5",
+        "knplabs/knp-menu-bundle": "^3.4",
+        "laminas/laminas-stdlib": "^3.19",
         "league/flysystem-bundle": "^3.3",
-        "lexik/jwt-authentication-bundle": "^2.11",
-        "liip/imagine-bundle": "^2.10",
-        "nyholm/psr7": "^1.6",
+        "lexik/jwt-authentication-bundle": "^2.21",
+        "liip/imagine-bundle": "^2.13",
         "pagerfanta/pagerfanta": "^4.0",
-        "payum/offline": "^1.7.3",
-        "payum/payum-bundle": "^2.5",
+        "payum/offline": "^1.7.5",
+        "payum/payum-bundle": "^2.6",
+        "php-http/discovery": "^1.20",
         "php-http/guzzle7-adapter": "^1.0",
         "php-http/httplug": "^2.4",
         "php-http/message-factory": "^1.0",
```

Comment:
Can we maybe get rid of this package? 

<img width="833" alt="Screenshot 2024-10-21 at 15 50 09" src="https://github.com/user-attachments/assets/f8cb7d06-2fd4-432b-81b4-3d45a93fc477">

Reply:
It doesn't seem to be that easy because of Payum, so for now, I've just moved it to dev dependencies 🖖🏻 
```
request.CRITICAL: Uncaught PHP Exception Symfony\Component\Messenger\Exception\HandlerFailedException: "Handling "Sylius\Bundle\PayumBundle\PaymentRequest\Command\CapturePaymentRequest" failed: No php-http message factories found. Note that the php-http message factories are deprecated in favor of the PSR-17 message factories. To use the legacy Guzzle, Diactoros or Slim Framework factories of php-http, install php-http/message and php-http/message-factory and the chosen message implementation." at DoctrineTransactionMiddleware.php line 42
```